liqd_product/assets/scss/components/_aside.scss

```diff
@@ -0,0 +1,17 @@
+.aside {
+    background-color: $bg-secondary;
+    color: contrast-color($bg-secondary);
+    padding: $padding;
+}
+
+.aside__block {
+    border-bottom: 1px solid $border-color;
+}
+
+.aside__block__title {
+    font-size: $font-size-sm;
+    text-transform: uppercase;
+    letter-spacing: 0.1em;
+    text-align: center;
+    font-weight: bold;
```

Comment:
That's why every heading level has a class too, so we can do something like `<h2 class="h3">`.